packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/src/config.ts

```diff
@@ -23,3 +28,33 @@ export const config: Config = {
    */
   port: process.env.PROXY_PORT || 3000,
 };
+
+export const personalizeConfig: PersonalizeConfig = {
+  // Configuration for your Sitecore Experience Edge endpoint
+  edgeConfig: {
+    clientFactory,
+    timeout:
+      (process.env.PERSONALIZE_MIDDLEWARE_EDGE_TIMEOUT &&
+        parseInt(process.env.PERSONALIZE_MIDDLEWARE_EDGE_TIMEOUT)) ||
+      400,
+  },
+  // Configuration for your Sitecore CDP endpoint
+  // Edge URL and ID can be taken from proxy env, or the base SPA app
+  cdpConfig: {
+    sitecoreEdgeUrl: process.env.SITECORE_EDGE_URL || fallbackEdgeEndpoint,
+    sitecoreEdgeContextId: process.env.SITECORE_EDGE_CONTEXT_ID || fallbackEdgeId || '',
```

Comment:
I respectfully disagree with this, but it is something we should discuss further as a separate talk.
For now this is a stopgap measure until we figure out the conflict between SPA and proxy env variables.

Reply:
so, correct - this thing is early to add, until we introduce support for clientside env variables
otherwise you can define some variables in proxy and in the browser you will pull values from SPA app env variables, this is the mismatch right now